src/Bravo3/ImageManager/Enum/ImageFormat.php

```diff
@@ -7,10 +8,12 @@
  * @method static ImageFormat PNG()
  * @method static ImageFormat JPEG()
  * @method static ImageFormat GIF()
+ * @method static ImageFormat PDF()
  */
 final class ImageFormat extends AbstractEnumeration
 {
     const PNG  = 'png';
     const JPEG = 'jpg';
```

Comment:
Not really used as an extension. This is used within DataInspector class for identifying JPEG compressed images. As far as I know it can be in either of these extensions .jpg or .jpeg